### PR TITLE
quick add custom domain 🔄

### LINKED ISSUE #2000: Custom autocomplete urls do not remain as https
_added by @bbinto_

**Why**
- Bug/issue: User expects that if they add an https website to FF's custom URLs, it should not request the http version later on.

**Acceptance Criteria (how do I know when I’m done?)**
- User adds HTTPS website to autocomplete list, when pulling it from the autocomplete list, it should keep the HTTPS

### PR DESCRIPTION
**Pull request by boek**
Closes #2000 #1933

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 1]
```diff
@@ -78,15 +80,18 @@
...
+import kotlin.jvm.functions.Function0;
 import mozilla.components.support.utils.ColorUtils;
 import mozilla.components.support.utils.DownloadUtils;
 import mozilla.components.support.utils.DrawableUtils;
 
 /**
  * Fragment for displaying the browser UI.
  */
-public class BrowserFragment extends WebFragment implements View.OnClickListener, DownloadDialogFragment.DownloadDialogListener {
+public class BrowserFragment extends WebFragment implements View.OnClickListener, DownloadDialogFragment.DownloadDialogListener, View.OnLongClickListener {
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[PMD\] INFO: Avoid really long classes\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[PMD\] INFO: The class 'BrowserFragment' has a Modified Cyclomatic Complexity of 4 \(Highest = 19\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[PMD\] INFO: The class 'BrowserFragment' has a NCSS line count of 536 \(Highest = 49\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[PMD\] INFO: The class 'BrowserFragment' has a Standard Cyclomatic Complexity of 4 \(Highest = 27\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[PMD\] INFO: The class 'BrowserFragment' has a total cyclomatic complexity of 78 \(highest 14\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[PMD\] INFO: This class has too many methods, consider refactoring it\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[PMD\] INFO: Too many fields

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 2]
```diff
@@ -1119,4 +1126,39 @@ public void onDismiss() {
     public void updateBlockingBadging(boolean enabled) {
         blockView.setVisibility(enabled ? View.GONE : View.VISIBLE);
     }
+
+    private void dismissAutocompletePopup() {
+        autocompletePopupWeakReference.get().dismiss();
+        autocompletePopupWeakReference.clear();
+    }
+
+    public boolean onLongClick(View view) {
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[PMD\] INFO: The method 'onLongClick\(View\)' has a NCSS line count of 16\.

**Comment by boek** [comment]
Looks like Sputnik doesn't like me editing BrowserFragment 😆

File: app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteQuickAddPopup.kt [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package org.mozilla.focus.autocomplete
```
Comment (ekager) [inline app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteQuickAddPopup.kt]: nit- license

File: app/src/main/res/layout/autocomplete_quick_add_popup.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ekager) [inline app/src/main/res/layout/autocomplete_quick_add_popup.xml]: license

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 5]
```diff
@@ -78,15 +80,20 @@
...
 import mozilla.components.support.utils.DownloadUtils;
 import mozilla.components.support.utils.DrawableUtils;
 
 /**
  * Fragment for displaying the browser UI.
  */
-public class BrowserFragment extends WebFragment implements View.OnClickListener, DownloadDialogFragment.DownloadDialogListener {
+@SuppressWarnings({"PMD.ExcessiveClassLength", "PMD.CyclomaticComplexity", "PMD.TooManyMethods",
+        "PMD.ModifiedCyclomaticComplexity", "PMD.TooManyFields", "PMD.StdCyclomaticComplexity" })
+public class BrowserFragment extends WebFragment implements View.OnClickListener, DownloadDialogFragment.DownloadDialogListener, View.OnLongClickListener {
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[PMD\] INFO: The class 'BrowserFragment' has a NCSS line count of 537 \(Highest = 49\)\.

File: app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteDomainFormatter.kt [hunk 6]
```diff
@@ -0,0 +1,14 @@
+package org.mozilla.focus.autocomplete
```
Comment (ekager) [inline app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteDomainFormatter.kt]: license?

**Pull request merged**